packages/firestore/test/unit/specs/write_spec.test.ts

```diff
@@ -645,7 +645,7 @@ describeSpec('Writes:', [], () => {
         expectRequestCount({ handshakes: 2, writes: 2, closes: 1 })
       )
       .expectNumOutstandingWrites(1)
-      .writeAcks('collection/key', 1, { expectUserCallback: false })
```

Comment:
Yeah, this was wrong. Disabling and re-enabling the network should not affect whether we resolve the user callback. This now fails since I am asserting that I didn't get any unexpected write acknowledgments.